scripts/deploy.py

```diff
@@ -1,16 +1,21 @@
 from pathlib import Path
 
-from brownie import Strategy, accounts, config, network, project, web3
+from brownie import AffiliateToken, Contract, accounts, config, network, project, web3
 from eth_utils import is_checksum_address
+
 import click
 
+# User Input 
+affiliateTokenName = "abcdef"
+affiliateTokenSymbol = "ABC"
```

Comment:
User should set this from `click.input`, but I am okay with this I guess

Reply:
Fixed it anyway